src/Compilers/CSharp/Portable/CommandLine/CSharpCommandLineParser.cs

```diff
@@ -1309,7 +1310,9 @@ public new CSharpCommandLineArguments Parse(IEnumerable<string> args, string bas
                 fileAlignment: fileAlignment,
                 subsystemVersion: subsystemVersion,
                 runtimeMetadataVersion: runtimeMetadataVersion,
-                instrumentationKinds: instrumentationKinds.ToImmutableAndFree()
+                instrumentationKinds: instrumentationKinds.ToImmutableAndFree(),
+                // TODO: set from /checksumalgorithm (see https://github.com/dotnet/roslyn/issues/24735)
```

Comment:
Why are we holding off on this part, seems important here?

Reply:
It is not that important. The algorithm used for PDB checksum in this PR is SHA256, which is good enough. Allowing to set the value to SHA384 or SHA512 will be important in future, but it can wait a bit. 

The reason to hold off is the default value of this switch. Right now it's SHA1, which is not acceptable for PDB checksum. The default has to be at least SHA256. If we want to use the same switch for both source checksums and PDB checksums then we need to set the default for source checksums to SHA256 first.